MAUI/LinearProgressBar/Animation.md

```diff
@@ -13,13 +13,13 @@ The linear progress bar provides animation support to visualize the progress val
 
 The following properties are used to define the duration of animation for the specific states.
 
-* `AnimationDuration:` Represents animation duration of the determinate state’s progress indicator.
-* `SecondaryAnimationDuration:` Represents animation duration of the determinate state’s secondary progress indicator.
-* `IndeterminateAnimationDuration:` Represents animation duration of the indeterminate state’s indicator.
+* [`AnimationDuration:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_AnimationDuration) Represents animation duration of the determinate state’s progress indicator.
+* [`SecondaryAnimationDuration:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfLinearProgressBar.html#Syncfusion_Maui_ProgressBar_SfLinearProgressBar_SecondaryAnimationDuration) Represents animation duration of the determinate state’s secondary progress indicator.
+* [`IndeterminateAnimationDuration:`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_IndeterminateAnimationDuration) Represents animation duration of the indeterminate state’s indicator.
 
 ## Easing effects
 
-The `AnimationEasing` property allows you specify the transfer function that controls animation speed when they run. 
+The [`AnimationEasing`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.ProgressBarBase.html#Syncfusion_Maui_ProgressBar_ProgressBarBase_AnimationEasing) property allows you specify the transfer function that controls animation speed when they run. 
 
 The following code sample demonstrates the `CubicInOut` easing function of the linear progress bar.
```

Comment:
API link not added.

Reply:
Since its a framework API we can't add the link.